app-scroll-effects/app-scroll-effects.html

```diff
@@ -1,29 +1,48 @@
-<!--
-@license
-Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
-This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
--->
-
-<link rel="import" href="app-scroll-effects-behavior.html">
-
-<script>
-
-(function(scrollEffects) {
-  function interpolate(progress, points, fn, ctx) {
-    fn.apply(ctx, points.map(function(point) {
-      return point[0] + (point[1] - point[0]) * progress;
-    }));
-  }
+function interpolate(progress, points, fn, ctx) {
+  fn.apply(ctx, points.map(function(point) {
+    return point[0] + (point[1] - point[0]) * progress;
+  }));
+}
+
+/**
+ * @constructor
+ * @extends Object
+ */
+var LocalDomWithBackground = function(){};
+/**
+ * @type HTMLElement
+ */
+LocalDomWithBackground.prototype.backgroundFrontLayer;
+/**
+ * @type HTMLElement
+ */
+LocalDomWithBackground.prototype.backgroundRearLayer;
+/**
+ * @type HTMLElement
+ */
+LocalDomWithBackground.prototype.background;
+
+/**
+ * @constructor
+ * @extends PolymerElement
+ */
+var ElementWithBackground = function(){};
+ElementWithBackground.prototype = {
+  $: new LocalDomWithBackground(),
+  isContentBelow: function(){},
+  _updateScrollState: function(){},
+  isOnScreen: function(){},
+  _deltaHeight: 0,
+}
```

Comment:
We plan to separate the effects into individual files and use this file to import all of them.